translate/convert/test_po2prop.py

```diff
@@ -363,6 +363,82 @@ def test_gaia_plurals(self):
         propfile = self.merge2prop(proptemplate, posource, personality="gaia")
         assert propfile == propexpected
 
+    def test_duplicates(self):
+        """Test back conversion of properties with duplicate units."""
+        # Test entries with same key and value.
+        proptemplate = '''
+key=value
+key=value
+'''
+        posource = r'''#: key
+msgid "value"
+msgstr "Waarde"
+'''
+        propexpected = '''
+key=Waarde
+key=Waarde
+'''
+        propfile = self.merge2prop(proptemplate, posource, personality="mozilla")
+        assert propfile == propexpected
+
+        # Test entries with same key and different value, and single
+        # corresponding entry in PO.
+        proptemplate = '''
+key=value
+key=another value
+'''
+        posource = r'''#: key
+msgid "value"
+msgstr "Waarde"
+'''
+        propexpected = '''
+key=Waarde
+key=Waarde
+'''
+        propfile = self.merge2prop(proptemplate, posource, personality="mozilla")
+        assert propfile == propexpected
+
+        # Test entries with same key and different value, and two different
+        # corresponding entries in PO.
+        proptemplate = '''
+key=value
+key=another value
+'''
+        posource = r'''#: key
+msgid "value"
+msgstr "Valor"
+
+#: key
+msgid "another value"
+msgstr "Outro valor"
+'''
+        propexpected = '''
+key=Valor
+key=Valor
+'''
+        propfile = self.merge2prop(proptemplate, posource, personality="mozilla")
+        assert propfile == propexpected
+
+        # Test entries with same key and different value.
+        proptemplate = '''
+key1=value
+key2=value
+'''
+        posource = r'''#: key1
```

Comment:
This is wrong, it wouldn't be a valid PO file.  prop2po would have added msgctxt.